contrib/mathjax-compat/README.md

```diff
@@ -0,0 +1,15 @@
+# MathJax Compatibility Extension
```

Comment:
It might make sense to mention that MathJax supports the same approach, but I agree that it should be called something more related to what it actually does.